From 4713fb985e14561c95c9171763849d8e7a0f4737 Mon Sep 17 00:00:00 2001
From: "Bandi, Kushal" <kushal.bandi@intel.com>
Date: Tue, 28 Aug 2018 19:19:33 -0700
Subject: [PATCH 2/2] media: intel-ipu4: [VIRT] Adding x86_64 arch dependency.

By adding this dependency ipu mediation is resctricted to
only x86_64 arch only.

Change-Id: I33dc975b17cc1fa9776d91d9f4c1d90af2116e29
Signed-off-by: Bandi, Kushal <kushal.bandi@intel.com>
---
 drivers/media/pci/intel/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/pci/intel/Kconfig b/drivers/media/pci/intel/Kconfig
index e328121..fca22a7 100644
--- a/drivers/media/pci/intel/Kconfig
+++ b/drivers/media/pci/intel/Kconfig
@@ -69,6 +69,7 @@ config VIDEO_INTEL_UOS
 	If selected UOS driver components will be compiled
 
 config VIDEO_INTEL_IPU_ACRN
+	depends on X86_64
 	bool "Compile for virtio mediation"
 
 choice
-- 
1.9.1

